### DISCUSSION TITLE
What's the purpose of the workspace?

### DISCUSSION THREAD

**Discussion by alexklibisz** [scala-steward-org/scala-steward#2817] [Q&A] [answered]
Scala-steward seems to run fine without it. Maybe it's just an optimization to minimize API calls?

**Comment by fthomas** [comment, marked as answer]
It is more than an optimization. Some features do not work if it is not persisted between runs. For example, closing obsolete PRs and the `pullRequests.frequency` features do not work without a workspace. See also https://github.com/scala-steward-org/scala-steward/blob/main/modules/docs/mdoc/running.md#workspace.

**Reply by alexklibisz** [reply]
Thanks! I should have kept reading those docs!
